samples/highcharts/website/xmas-2021/demo.js

```diff
@@ -0,0 +1,3937 @@
+            {
+                type: 'line',
+                zIndex: 100,
+                visible: false,
+                enableMouseTracking: false,
+                lineWidth: 2,
+                xAxis: 4,
+                yAxis: 4,
+                color: '#BD8032',
+                dragDrop: {
+                    draggableX: true,
+                    draggableY: true
+                },
+                marker: {
+                    enabled: false
+                },
+                data: [{
+                    x: 3.22,
+                    y: 5.46
+                },
+                {
+                    x: 17,
+                    y: 5.46
+                }]
+            },
+            //6 scatter bow
+            {
+                type: 'scatter',
+                visible: false,
+                zIndex: 100,
+                lineWidth: 2,
+                lineColor: 'transparent',
+                xAxis: 4,
+                yAxis: 4,
+                color: '#FF4540',
+                marker: {
+                    enabled: true,
+                    symbol: 'triangle',
+                    radius: 10
+                },
+                data: [{
+                    x: 9.5,
+                    y: 19.2
+                },
+                {
+                    x: 10.5,
+                    y: 18,
+                    marker: {
+                        enabled: true,
+                        symbol: 'circle',
+                        radius: 8
+                    }
+                },
+                {
+                    x: 11.5,
+                    y: 19.2
+                }]
+            }]
+    });
+};
+
+
+const tree = function () {
+    Highcharts.chart('tree-chart', {
+
+        chart: {
+            type: 'columnrange',
+            margin: 0,
+            backgroundColor: 'transparent',
+            inverted: true,
+            events: {
+                load: function () {
+                    const chart = this;
+
+                    let count = 0;
+                    const treeData =  [
+                        [10, 10],
+                        [9, 14],
+                        [6, 17],
+                        [3, 20],
+                        [0, 23],
+                        [-3, 26],
+                        [-7, 29],
+                        [-10, 32],
+                        [7, 17],
+                        [7, 17],
+                        [7, 17],
+                        [10, 10],
+                        [0, 0]
+                    ];
+
+                    const topTree = document.querySelector('#tree-chart .highcharts-series-2');
+
+                    chart.series[0].update({
+                        color: '#E05283'
+                    });
+
+                    chart.series[3].data.forEach(function (point) {
+                        point.update({
+                            low: -30,
+                            high: 60
+                        });
+                    });
```

Comment:
As far as I can see, `series[3]` has multiple data points, so in this case we should call `point.update` with the `redraw` parameter set to false, then call `chart.redraw()` after the loop.